### ISSUE TITLE
uninitialized constant Shellwords (NameError)

### ISSUE THREAD

**Issue by kylef** [supermarin/xclisten#15]
Perhaps a missing or incorrect dependency version?

``` bash
$ xclisten -s FitnessFirstTests
/Library/Ruby/Gems/2.0.0/gems/xclisten-0.0.7/bin/xclisten:25:in `block (2 levels) in <top (required)>': uninitialized constant Shellwords (NameError)
    from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/optparse.rb:1395:in `call'
    from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/optparse.rb:1395:in `block in parse_in_order'
    from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/optparse.rb:1351:in `catch'
    from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/optparse.rb:1351:in `parse_in_order'
    from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/optparse.rb:1345:in `order!'
    from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/optparse.rb:1437:in `permute!'
    from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/optparse.rb:1459:in `parse!'
    from /Library/Ruby/Gems/2.0.0/gems/xclisten-0.0.7/bin/xclisten:32:in `block in <top (required)>'
    from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/optparse.rb:885:in `initialize'
    from /Library/Ruby/Gems/2.0.0/gems/xclisten-0.0.7/bin/xclisten:13:in `new'
    from /Library/Ruby/Gems/2.0.0/gems/xclisten-0.0.7/bin/xclisten:13:in `<top (required)>'
    from /usr/bin/xclisten:23:in `load'
    from /usr/bin/xclisten:23:in `<main>'
```

**Comment by supermarin** [comment]
@kylef thanks for the report Kyle!
It seems like some rubies need explicit `require 'shellwords'` and some don't.
Gonna release the fix today

**Comment by keith** [comment]
I see this as well when using either of those flags. I fixed with just an import in `bin/xclisten`

**Comment by supermarin** [comment]
Fixed / released in 0.0.8
